tools/python/runfiles/runfiles_constants.py

```diff
@@ -0,0 +1,2 @@
+# Only used by tests.
+MAIN_REPOSITORY_RUNFILES_DIRECTORY = "io_bazel"
```

Comment:
The presence of this file is a bit confusing. So there's a generated one and fake one? Why does this fake one need to exist?

Reply:
I improved the BUILD setup so that the test now also uses the generated file.